### PR TITLE
Update to clojure 1.9.0, fixes specs, add s/keys coercion

### PR DESCRIPTION
**Pull request by joodie**
Hi, I was playing around with your awesome work and I was missing an s/keys coercion, so I added that. I also ran into lots of errors running the tests with spec instrumentation on, so I removed a whole bunch of specs that seemed to be erroneously strict, so now the tests run with instrumentation on.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(defproject spec-coerce "1.0.0-alpha6-SNAPSHOT"
+(defproject nl.zeekat/spec-coerce "1.0.0-alpha6-SNAPSHOT"
```
Comment (wilkerlucio) [inline project.clj]: can you please revert the project name?

**Comment by wilkerlucio** [comment]
Ah, one more question, about the `s/keys`, had you checked the `coerce-structure` function? I just remembered I hadn't added `s/keys` because the structure should suffice that case, so I wonder if you have a new case where it doesn't work, I would like to understand that.

**Comment by joodie** [comment]
The case I was working with is where I already have a spec for a map for validation, and then I have a few different input methods, some of which only support string as values. using coerce-structure would require me to basically duplicate the s/keys spec.

**Comment by wilkerlucio** [comment]
Cool, thanks for the clarification.

**Comment by wilkerlucio** [comment]
@joodie `[spec-coerce "1.0.0-alpha6"]` is released on Clojars.

**Comment by joodie** [comment]
@wilkerlucio great, thanks!

**Pull request merged**